### DISCUSSION TITLE
Achievements

### DISCUSSION THREAD

**Discussion by itsgareth** [community/community#19473] [Other Feature Feedback, Questions, & Ideas]
### Here are my ideas:
- ### Visual
  - [x] Make them smaller
  - [x] Remove the labels 
  - [x] Use the metallic badge logo
- ### Options
  - [x] Enable / Disable specific achievements
  - [x] Change the display order 

## Before and After
<div>
<img align="left" height="600" alt="Screenshot 2022-06-26 at 09 45 29" src="https://user-images.githubusercontent.com/34524710/175806713-f4c934cf-143d-45c0-b5af-5d03fdfab290.png">
<img align="right" height="600" alt="Screenshot 2022-06-26 at 09 47 20" src="https://user-images.githubusercontent.com/34524710/175806717-75746483-11e2-43f9-849a-09645936b7ad.png">
</div>

**Comment by melvincwng** [comment]
@gwdawson, great suggestion!

I do agree on the usage of the metallic badge logos. Using numbers like x2, x3, x4 to denote the tiers of your achievements, can be a bit confusing, especially to newcomers.

**Reply by itsgareth** [reply]
absolutely! i think the current logos just look a little too childish

**Comment by 0xd3bu9** [comment]
What if to used to use a metal badge and a laurel wreath underneath?

**Comment by nlavanderos** [comment]
<3
